### ISSUE TITLE
Incompatible dependencies in pipenv install

### ISSUE THREAD

**Issue by joshuagruenstein** [rlworkgroup/garage#1758]
Hello,

When I try to install garage in a fresh pipenv as per the documentation, I get the following error:

```
[pipenv.exceptions.ResolutionFailure]: Warning: Your dependencies could not be resolved. You likely have a mismatch in your sub-dependencies.
  First try clearing your dependency cache with $ pipenv lock --clear, then try the original command again.
 Alternatively, you can use $ pipenv install --skip-lock to bypass this mechanism, then run $ pipenv graph to inspect the situation.
  Hint: try $ pipenv lock --pre if it is a pre-release dependency.
ERROR: Could not find a version that matches cloudpickle==1.3,~=1.2.0 (from garage==2020.6.0->-r /tmp/pipenvprlocesvrequirements/pipenv-7gor6s43-constraints.txt (line 2))
Tried: 0.1.0, 0.1.0, 0.1.1, 0.1.1, 0.2.1, 0.2.1, 0.2.2, 0.2.2, 0.3.0, 0.3.0, 0.3.1, 0.3.1, 0.4.0, 0.4.0, 0.4.1, 0.4.1, 0.4.2, 0.4.2, 0.4.3, 0.4.3, 0.4.4, 0.4.4, 0.5.0, 0.5.0, 0.5.1, 0.5.1, 0.5.2, 0.5.2, 0.5.3, 0.5.3, 0.5.4, 0.5.4, 0.5.5, 0.5.5, 0.5.6, 0.5.6, 0.6.0, 0.6.0, 0.6.1, 0.6.1, 0.7.0, 0.7.0, 0.8.0, 0.8.0, 0.8.1, 0.8.1, 1.0.0, 1.0.0, 1.1.1, 1.1.1, 1.2.0, 1.2.0, 1.2.1, 1.2.1, 1.2.2, 1.2.2, 1.3.0, 1.3.0, 1.4.0, 1.4.0, 1.4.1, 1.4.1, 1.5.0, 1.5.0
There are incompatible versions in the resolved dependencies:
  cloudpickle (from garage==2020.6.0->-r /tmp/pipenvprlocesvrequirements/pipenv-7gor6s43-constraints.txt (line 2))
  cloudpickle==1.3 (from tensorflow-probability==0.10.1->garage==2020.6.0->-r /tmp/pipenvprlocesvrequirements/pipenv-7gor6s43-constraints.txt (line 2))
  cloudpickle~=1.2.0 (from gym[atari,box2d,classic_control]==0.15.4->garage==2020.6.0->-r /tmp/pipenvprlocesvrequirements/pipenv-7gor6s43-constraints.txt (line 2))
```
This can be shortcutted by appending `--skip-lock` to the `pipenv install`, but that's obviously not ideal.  Thanks!

**Comment by ryanjulian** [comment]
Hey @joshuagruenstein, thanks for trying garage and reporting the issue.

* Are you installing garage from PyPI or source?
* Which version of Python is your pipenv based on?
* Which version of pipenv are you using?

**Comment by joshuagruenstein** [comment]
Hi!  I'm using pipenv version 2020.6.2, Python 3.6.9, and installing from PyPI.

**Comment by ryanjulian** [comment]
@joshuagruenstein This is now fixed on master (and was actually already fixed in the release branch, but we hadn't cut bugfix release yet).

You should see the fix appear on PyPI as `v2020.06.1` later today.
